docs/howto/update_major_package_spec.md

```diff
@@ -59,3 +59,32 @@ some of the expected values.
 
 For example if a document contains `event.category: web`, the value of
 `event.type` must be `access`, `error` or `info` according to ECS 8.4.
+
+### field "event.dataset" should have value ..., it has ...
+
+The fields `event.dataset` and `data_stream.dataset` should contain the name of
+the package and the name of the data stream that generates it, separated by a
+dot. For example for documents of the "access" data stream of the Apache module,
+it should be `apache.access`.
```

Comment:
In this section, I think it should also be added `data_stream.dataset`. Same error could happen in that field.

```suggestion
### field "event.dataset"/"data_stream.dataset" should have value ..., it has ...

Both fields `event.dataset` and `data_stream.dataset` should contain the name of
the package and the name of the data stream that generates it, separated by a
dot. For example for documents of the "access" data stream of the Apache module,
it should be `apache.access`.
```

Reply:
Ignore this comment, it was before you pushing the latest changeset.